### ISSUE TITLE
batchsize issue

### ISSUE THREAD

**Issue by superzrx** [lim0606/torch-inception-resnet-v2#3]
according to "We have trained our networks with stochastic gradient
utilizing the TensorFlow [1] distributed machine learning
system using 20 replicas running each on a NVidia Kepler
GPU" from paper
maybe the effective batchsize is 20*32
